lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -222,8 +222,9 @@ def write(chunk)
           'entries' => [],
         }
         arr.each do |time, record|
```

Comment:
Where is "time" taken from if there is nothing that can used in the record?
Also I think the original report was that "timestamp" was a "seconds since epoch", and is that reasonable that it should also work? Or in that case we expect the user to use a combo of "timestampSeconds" and "timestampNanos" in the record?